packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -148,6 +147,8 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
             }
             key={ selectedIndex }
             aria-atomic={ true }
+            role='textbox'
+            aria-readonly='true'
```

Comment:
If we want to overhaul dropdown then we can to allow it to be editable (allowFreeform) as well as autoComplete then we can do that. We'd also need to think about how to handle the fact that there's no placeholder for combobox and onRenderTitle/onRenderPlaceholder introduce broken concepts for the comboBox